shared/common-adapters/plain-input.stories.js

```diff
@@ -46,6 +130,9 @@ const load = () => {
     .add('With placeholder color', () => (
       <PlainInput {...commonProps} placeholder="I'm blue!" placeholderColor="blue" />
     ))
+
+  // Sandbox for testing controlled input bugginess
+  storiesOf('Common', module).add('Controlled input playground', () => <ControlledInputPlayground />)
```

Comment:
Cool! The main thing I wanted to test, though, was something that would insert/replace text. See the Input story 'Empty (uncontrolled)' and 'Empty (multiline) (uncontrolled)' for what I mean. Can you add a story like that? (Then it'll probably make it clear what I mean about being confused by the API...)

Reply:
Added a replica of `Empty (uncontrolled)`